exercises/src/test/scala/fpinscala/LazinessSpec.scala

```diff
@@ -0,0 +1,153 @@
+package fpinscala
```

Comment:
nit: Normally, the tests should be in the same package as the code that's being tested. So in this case, `fpinscala.laziness`. Being in the same package means less imports and makes it easier to access package protected methods.